src/common/catalog/catalog-entity-registry.ts

```diff
@@ -13,7 +18,7 @@ export class CatalogEntityRegistry {
   }
 
   @computed get items(): CatalogEntity[] {
-    return Array.from(this.sources.values()).flat();
+    return Array.from(iter.flatMap(this.sources.values(), source => source.get()));
```

Comment:
I don't think this specific name change is very good, because we are flat mapping over `sources.values()`. Also `sources` shouldn't really be called `entities` because it is a map between IDs and `IComputedValue<CatalogEntity[]>`.

Reply:
I guess conceptually it is a source but it is confusing for me because there are two different data types you are calling "source". You're not iterating over the `this.sources` map, but rather `this.sources.values()`, which gives you an iterable over just the entities, free of the IDs, right? I still like `entities` (not `entity`, my mistake) better here than `source`. Anyway, it's fine

Reply:
Yes I am iterating over `this.sources.values()` but that is of the type `Iterable<CatalogEntities[]>` which is why I do a `iter.flatMap`